### ISSUE TITLE
Training face detection model (starting from pre-trained weights)

### ISSUE THREAD

**Issue by manisoftwartist** [open-edge-platform/geti#633]
Hi, I am trying to retrain `face-detection-0204` and had a few clarifications:

1. `mobilenetv2_w1` seems to to be the base network (for face-detection-0204 as well as several other models) - But where is the network defined? 
2. The pre-trained weights in `snapshot.pth` are obtained by training on which dataset? 
3. I am trying to retrain `face-detection-0204` on the `WIDERFace` training dataset. After instantiating the template for 0204, for the retraining. I use 
`python train.py --load-weights snapshot.pth` pointing the other parameters to the WIDERFace dataset. I use the same learning rate of 0.05 as in the template.
But I am afraid the training is not working. I see the mAP increasing and the training loss saturating. Please see my training graphs below:
![Screenshot 2021-10-11 at 6 20 15 PM](https://user-images.githubusercontent.com/707881/136793086-a5ef98d5-21b4-435a-97fc-b52f1afabc04.png)

If I set Smoothing to 0 in tensorboard, the loss is simply noise. I dont think the model is learning anything at all !!
![Screenshot 2021-10-12 at 7 52 31 AM](https://user-images.githubusercontent.com/707881/136880535-22a2bdcb-0d7d-41e2-932a-114e16e2f33e.png)
The mAP is decreasing on `WIDER_val` which I have provided as validation set!
![Screenshot 2021-10-11 at 6 20 34 PM](https://user-images.githubusercontent.com/707881/136793313-fe7177ea-d323-4d30-a139-4612929929a2.png)

The learning rate is constant:
![Screenshot 2021-10-11 at 6 20 20 PM](https://user-images.githubusercontent.com/707881/136793133-eab2eb09-3da0-4126-b624-630aa55e43c9.png)

Do you have any suggestions on how I go about debugging this?
